src/index.js

```diff
@@ -1,49 +1,192 @@
 const {
   BaseKonnector,
   requestFactory,
-  saveFiles,
-  addData
+  saveBills,
+  updateOrCreate,
+  scrape,
+  signin
 } = require('cozy-konnector-libs')
-const request = requestFactory({ cheerio: true })
+const log = require('cozy-logger')
+const pdf = require('pdfjs')
+const html2pdf = require('./html2pdf')
+const moment = require('moment')
 
-const baseUrl = 'http://books.toscrape.com'
+const baseUrl = 'https://www.netflix.com'
+const NBSPACE = '\xa0'
+const DEBUG = false
+
+process.env.SENTRY_DSN =
+  process.env.SENTRY_DSN ||
+  'https://397c3282a3bd46b39eaf5e91770fe362:6f81483e6c1e4760a3a69a937fecf147@sentry.cozycloud.cc/44'
 
 module.exports = new BaseKonnector(start)
 
-// The start function is run by the BaseKonnector instance only when it got all the account
-// information (fields). When you run this connector yourself in "standalone" mode or "dev" mode,
-// the account information come from ./konnector-dev-config.json file
-function start(fields) {
-  // The BaseKonnector instance expects a Promise as return of the function
-  return request(`${baseUrl}/index.html`).then($ => {
-    // cheerio (https://cheerio.js.org/) uses the same api as jQuery (http://jquery.com/)
-    // here I do an Array.from to convert the cheerio fake array to a real js array.
-    const entries = Array.from($('article')).map(article =>
-      parseArticle($, article)
-    )
-    return addData(entries, 'io.cozy.books').then(() =>
-      saveFiles(entries, fields)
-    )
+async function start(fields) {
+  const request = requestFactory({ cheerio: true, jar: true })
+  const { email, password } = fields
+  // follow some redirects to get correct cookie & login urls
+  const resp = await request({
+    url: `${baseUrl}/login`,
+    debug: DEBUG,
+    resolveWithFullResponse: true
+  })
+
+  const $ = await signin({
+    url: resp.request.uri.href,
+    debug: DEBUG,
+    formSelector: 'form.login-form',
+    formData: { email, password },
+    validate: (status, $) => {
+      return status == 200 && $('.list-profiles').length == 1
+    },
+    headers: {
+      'Accept-Language': 'en-US'
+    },
+    followRedirect: true
   })
+
+  await selectProfile($, fields)
+
+  // we need the build number to make API calls
+  const buildNumber = await getBuildNumber($)
```

Comment:
Must have been quite funny to grasp... :muscle: :ok_hand:

Reply:
Got lucky, it changed in the middle of development :-)

Reply:
Oh yeah :sunglasses: